trace/Sampling.md

```diff
@@ -8,7 +8,31 @@ exporters.
 
 The Sampling bit is always set only at the start of a Span, using a `Sampler`
 
-### What kind of samplers does OpenCensus support?
+### Sampler interface
+A `Sampler` is an interface that returns a `SamplingDecision` based on given sampling parameters.
+
+#### Sampling parameters
+Based on the language the sampling parameters can be embedded into an object called
+`SamplingParameters` or explicitly passed as arguments. The set of sampling parameters SHOULD
+include at least:
+* The parent `SpanContext`.
+* The current `TraceId`.
+* The current `SpanId`.
+* The current `Tracestate`.
+* The current `Span`'s name.
+* The current `Span`'s kind.
+
+#### SamplingDecision
+`SamplingDecision` is the object returned by the `Sampler` and SHOULD include:
+* A boolean value that represents the sampling decision.
+* A list of `Attribute`'s that are added to the newly created `Span`.
```

Comment:
have you meant that WILL be added? or that ARE added?

Reply:
Done.